cli/internal/k8s/pods.go

```diff
@@ -2,17 +2,80 @@ package k8s
 
 import (
 	"context"
-	"github.com/defenseunicorns/zarf/cli/types"
 	"sort"
 	"time"
 
+	"github.com/defenseunicorns/zarf/cli/types"
+
 	"github.com/defenseunicorns/zarf/cli/internal/message"
 	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
 const waitLimit = 30
 
+func GeneratePod(name string, namespace string) *corev1.Pod {
+	message.Debugf("k8s.GeneratePod(%s, %s)", name, namespace)
+	return &corev1.Pod{
+		TypeMeta: metav1.TypeMeta{
+			APIVersion: corev1.SchemeGroupVersion.String(),
+			Kind:       "Pod",
+		},
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      name,
+			Namespace: namespace,
+			Labels: map[string]string{
+				// track the creation of this ns by zarf
+				"app.kubernetes.io/managed-by": "zarf",
```

Comment:
I've seen this same string a few times now. Worth making more DRY?

Reply:
Yeah I think so